pages/browse/index.vue

```diff
@@ -2,12 +2,54 @@
 const { t } = useI18n();
 toolbarTitleStore().setReactiveToolbarTitle(() => t("nav.browse"));
 
-const { data: playlists } = useCuratedPlaylists();
+const { data: browseSections, pending } = await useBrowse();
 </script>
 
 <template>
   <div>
     <PageHeading class="mb-6">{{ $t("nav.browse") }}</PageHeading>
-    <PlaylistOverview v-if="playlists" :playlists="playlists" />
+    <template v-if="pending">
+      <ul>
+        <li
+          v-for="index in 5"
+          :key="index"
+          class="my-6 h-11 w-full animate-pulse rounded-lg bg-background-2"
+        ></li>
+      </ul>
+    </template>
+    <ul v-else>
+      <template v-for="section in browseSections" :key="section.id || 0">
+        <PageHeading v-if="section.type === 'section_header'" :level="3">
+          <div class="flex items-center justify-between">
+            <div>
+              <NuxtLink
+                v-if="typeof section.link === 'string'"
+                :to="section.link as any"
```

Comment:
Do we need this `as any` here? The nuxt-link seems to anyways have some trouble with external links in the desktop-app, since they're external links.

And - wouldn't this be a duplicate of the next `a`-tag?

Reply:
This is only done because I wanted to use `NuxtLink` here incase we would use relative link paths, but if we won't I'll just change this back to an `a` tag.